extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -137,6 +140,8 @@ class AmpStickyAd extends AMP.BaseElement {
           // now.
           this.vsync_.mutate(() => {
             this.element.classList.add('amp-sticky-ad-loaded');
+            this.closeButton_.classList.add
+                ('amp-sticky-ad-close-button-loaded');
```

Comment:
This is redundant. They can use `.amp-sticky-ad-loaded .amp-sticky-ad-close-button` to get the button once the ad is "loaded".